README.md

```diff
@@ -79,6 +79,7 @@
 - [vcli](https://github.com/changhz/vcli) - A CLI tool to generate folder structure according to the [guideline](https://blog.vlang.io/the-complete-beginners-guide-to-cli-apps-in-v/)
 - [vgoogle](https://github.com/changhz/vgoogle) - Make google search on the terminal.
 - [vzcc](https://github.com/malisipi/vzcc) - A CLI cross-compiling tool based on Zig CC for V.
+- [vindex](https://github.com/wenxuanjun/vindex) - A simple file list server generating json strings, compatible with nginx's autoindex module.
```

Comment:
Please add items in alphabetical order.

Reply:
This entry should be between `vgoogle` and `vzcc`.

Reply:
Sorry for not noticing that. It's now fixed and thanks a lot.